### PR TITLE
Preview of documents to be added as part of ASWF transfer

### PR DESCRIPTION
**Pull request by lgritz**
Including new Technical Charter, governance documents, and CLAs.

These are all drafts, subject to review and vote by the TSC and not in
effect until that happens (at which point they will be moved around
the repo, no longer treated as drafts, and in some cases like the CLAs
will replace entirely the prior versions that will no longer be
relevant).

They are as identical as possible to the corresponding documents of
OpenColorIO (differing only for name changes and the like, and in just one or
two places, substituting an equivalent but slightly more clearly
worded version of some piece from OpenEXR). None of the important
legal stuff like CLAs or licenses have been altered from the OCIO and
ASWF standard template as they were approved (by my organization) for
that project.

The license remains the 3-clause BSD we have always used, so there is
no reason to contact prior authors for re-licensing. But when we get
around to moving the repo from the Imageworks GitHub account to the
ASWF one, the new CLAs will take effect and subsequent contributors
will need to sign them (even if they had previously signed the Sony
ones).

### CODE REVIEW

File: ASWF/CLA-corporate.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+Corporate Contributor License Agreement ("Agreement")
+
+Thank you for your interest in the Open Shading Language Project a Series of
+LF Projects, LLC (hereinafter "Project"). In order to clarify the
+intellectual property licenses granted with Contributions from any corporate
+entity to the Project, LF Projects, LLC ("LF Projects") is required to have
+a Corporate Contributor License Agreement (CCLA) on file that has been
+signed by each contributing corporation.
```
Comment (jmertic) [inline ASWF/CLA-corporate.md:8]: This section should read....

```
Thank you for your interest in the Open Shading Language Project (hereinafter
 "Project"). In order to clarify the intellectual property licenses granted with 
Contributions from any corporate entity to the Open Shading Language Project 
a Series of LF Projects, LLC ("LF Projects") is required to have
a Corporate Contributor License Agreement (CCLA) on file that has been
signed by each contributing corporation.
```

File: ASWF/CLA-individual.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
+Individual Contributor License Agreement ("Agreement")
+
+Thank you for your interest in the Open Shading Language Project a Series of
+LF Projects, LLC (hereinafter "Project"). In order to clarify the
+intellectual property licenses granted with Contributions from any corporate
+entity to the Project, LF Projects, LLC ("LF Projects") is required to have
+an Individual Contributor License Agreement (ICLA) on file that has been
+signed by each contributing individual. (For legal entities, please use the
+Corporate Contributor License Agreement (CCLA).)
```
Comment (jmertic) [inline ASWF/CLA-individual.md:9]: Same as above - first reference `Open Shading Language Project` and second `Open Shading Language Project a Series of LF Projects, LLC`

File: ASWF/DCO.md [hunk 3]
```diff
@@ -0,0 +1 @@
+https://developercertificate.org/
```
Comment (jmertic) [inline ASWF/DCO.md:1]: You could also link to https://github.com/AcademySoftwareFoundation/tac/blob/master/process/contributing.md#contribution-sign-off

File: ASWF/CLA-individual.md (hunk 2 above)
Comment (lgritz) [reply ASWF/CLA-individual.md:9]: OK, thanks, I will make those edits.
I thought I copied these straight from other projects, so you may want to check openexr and ocio. I could have botched it, or those may also be the same.

File: ASWF/DCO.md (hunk 3 above)
Comment (lgritz) [reply ASWF/DCO.md:1]: You mean from the CONTRIBUTING, and not have the stub DCO.md file at all?

File: ASWF/CLA-corporate.md (hunk 1 above)
Comment (lgritz) [reply ASWF/CLA-corporate.md:8]: I pushed an update implementing these instructions.

File: ASWF/CLA-individual.md (hunk 2 above)
Comment (lgritz) [reply ASWF/CLA-individual.md:9]: I pushed an update implementing these instructions.

File: ASWF/DCO.md (hunk 3 above)
Comment (jmertic) [reply ASWF/DCO.md:1]: That works too - just saw this was in here and was trying to understand the intention.

Comment (lgritz) [reply ASWF/DCO.md:1]: The intention was to copy verbatim what I saw in OpenColorIO.

Comment (lgritz) [reply ASWF/DCO.md]: I'm just going to eliminate the DCO file, leave the instructions I added to CONTRIBUTING.

File: ASWF/CLA-corporate.md (hunk 1 above)
Comment (jmertic) [reply ASWF/CLA-corporate.md]: Perfect - thanks! We are getting in place these agreements for EasyCLA now.

Comment (jmertic) [reply ASWF/CLA-corporate.md]: Ok - you're gonna kill me - but let's revert back to what you originally provided to make this consistent with other projects. Sorry :(

File: ASWF/CLA-individual.md (hunk 2 above)
Comment (jmertic) [reply ASWF/CLA-individual.md]: Ok - you're gonna kill me - but let's revert back to what you originally provided to make this consistent with other projects. Sorry :(

Comment (lgritz) [reply ASWF/CLA-individual.md]: hahahaha
no problem

File: ASWF/CLA-corporate.md [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+
+Each contributing corporation ("You") must accept and agree that, for any
+Contribution (as defined below), You and all other individuals and entities
+that control You, are controlled by You, or are under common control with
+You, are bound by the licenses granted and representations made as though
```
[the remark this answers is not in the document]
Comment (lgritz) [reply ASWF/CLA-corporate.md:13]: This is the same as the master ASWF template and the OCIO and OpenEXR CLAs (I'm pretty sure Pixar has signed the OpenEXR one, right?).

[the remark this answers is not in the document]
Comment (lgritz) [reply ASWF/CLA-corporate.md:13]: Bouncing this to @jmertic. This is above my pay grade. 

Our goal here was to use the existing CLAs, which my org has already approved for this and previous projects. On prior projects' entrance to the ASWF, these were negotiated to death by the various parties, and I truly dread having to ask them to enter additional rounds of revision with new language.

My advice is that if a company thinks the ASWF CLAs should be worded differently, that is something they should address or petition to change through the Governing Board and its legal committee.

[the remark this answers is not in the document]
Comment (sfriedmapixar) [reply ASWF/CLA-corporate.md:13]: It does seem reasonable to surmise that if they're the same it should be OK, but it seems that legal's experience with the OpenEXR CLA is what prompted the suggestion.

[the remark this answers is not in the document]
Comment (jmertic) [reply ASWF/CLA-corporate.md:13]: I'd agree that this is not the right forum if one's org has concerns @lgritz 

@sfreidmapixar - if it makes sense let's connect LF legal with your legal offline - can you drop me a note at jmertic@linuxfoundation.org?

**Comment by lgritz** [comment]
Updated this PR:

* TSC unanimously approved the governance and charter.
* Rename the CLA files to clearly indicate that they are drafts, still some haggling going on.

**Pull request merged**